Resources/Prototypes/Entities/Objects/Consumable/Food/meat.yml

```diff
@@ -1211,3 +1211,58 @@
           Quantity: 1
         - ReagentId: Protein
           Quantity: 1
+
+- type: entity
+  name: raw shrimp
+  parent: FoodMeatBase
+  id: FoodMeatShrimp
+  description: You can barbeque it, boil it, broil it, bake it, saute it...
+  components:
+  - type: FlavorProfile
+    flavors:
+      - fishy
+  - type: Tag
+    tags:
+    - Raw
+  - type: Sprite
+    state: shrimp
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        reagents:
+        - ReagentId: CarpoToxin
+          Quantity: 0.5
+  - type: Extractable
+    juiceSolution:
+      reagents:
+      - ReagentId: CarpoToxin
+        Quantity: 0.5
+      - ReagentId: Brine
```

Comment:
I need to remove this, i decided not to make a brine reagent since i would have to make some unique ingredient to go along with it or it isnt worth time.